src/main/java/com/zendesk/maxwell/schema/Table.java

```diff
@@ -144,6 +117,15 @@ private void renumberColumns() {
 			c.setPos(i++);
 		}
 	}
+
+	public void setDefaultColumnEncodings() {
```

Comment:
nitpic: Just noticing an inconsistency here... Table refers to its database and falls back to its encoding. But column does not refer to its table. And table forces its default encoding wherever columns are changed or added.